Overview.bs

```diff
@@ -676,19 +677,25 @@ transferred:
 
 ### Extending the Font Subset ### {#extend-subset}
 
-A client extends its font subset to cover additional codepoints by making HTTP requests to
-a Patch Subset server. The HTTP request must use either the GET or POST method:
+A client extends its font subset to cover additional codepoints by making requests to a Patch Subset
+server. The request requirements are described here in terms of a fetch according to
+"4. Fetching" in [[!fetch]]. If the implementing user agent does not support fetch, then the request
+should be made using an equivalent HTTP request.
 
-*  If sent as a POST request the post body will be a single
-    <a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR.
+* The request <strong>method</strong> must be either "GET" or "POST".
```

Comment:
This is Bikeshed (https://tabatkins.github.io/bikeshed/), so you should use its cross-spec referencing abilities:

```
* The request [=request/method=] must be either "GET" or "POST".
```